wearerequired/traduttore
# Add filter to the entries Export status

**Description**

This allows to set a custom set of statuses, like 'current_or_untranslated'.

Fix #259 

**How has this been tested?**
1. Set a different value on the status filter.
2. Check out the language packs as desired.

**Types of changes**
New feature (non-breaking change which adds functionality)

**Checklist:**
